src/test/java/LibraryTest.java

```diff
@@ -0,0 +1,59 @@
+import com.mvelyka.library.Book;
+import com.mvelyka.library.Library;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class LibraryTest {
+
+    Library library;
+    Book[] books;
+
+    @Test
+    public void borrowBookTrueTest() {
```

Comment:
The naming is not good. We can use something like:
shouldBorrowAvailableBook
shouldNotBorrowNotAvailableBook

It's more clear